src/main/java/com/softech/coursesvc/controller/CourseController.java

```diff
@@ -1,17 +1,39 @@
 package com.softech.coursesvc.controller;
 
+import com.softech.coursesvc.CourseList;
+import com.softech.coursesvc.entity.CourseEntity;
+import com.softech.coursesvc.service.CourseSvc;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+import java.util.List;
+
 @RestController
 public class CourseController {
+    private final CourseList courseList;
```

Comment:
Why making it a global variable.. 
You can declare it inside method where you are creating it as local variavle..if you do this.. you are making it final ..it will become final.. renove it from here

Reply:
Done. I am making changes now.